crates/serpen/tests/fixtures/ast_rewriting/class_name_collision/main.py

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+"""
+Class name collision test.
+Both modules define classes with the same names.
+"""
+
+from models import User as ModelUser, Product as ModelProduct
+from entities import User as EntityUser, Product as EntityProduct
+
+
+def main():
```

Ruff (0.11.9)

Missing return type annotation for public function `main`

(ANN201)